### PR TITLE
[Task CS-20] Added files parser

### PR DESCRIPTION
**Pull request by twistzzmc**

### CODE REVIEW

**Review by jmysliv — COMMENTED** [review]
Fix pylint errors

**Review by jmysliv — APPROVED** [review]
lgtm

File: rest/common/parsing.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    :param file: .csv file to be parsed
+    :return: list of pairs (document type, list of dictionaries),
+        where in each of the dictionary there is one candidate/recruitment
+    """
+    data = []
+    document_type = None
+    columns = []
+
+    with open(file) as csv_file:
+        csv_reader = csv.reader(csv_file, delimiter=',')
```
Comment (twrdyyy) [inline rest/common/parsing.py:88]: I think we should point that delimiter is ","

**Review by twrdyyy — APPROVED** [review]

File: rest/common/parsing.py (hunk 1 above)
Comment (twistzzmc) [reply rest/common/parsing.py:88]: In docstring you mean?

Comment (twrdyyy) [reply rest/common/parsing.py:88]: in generator we have specified "," as delimiter as well but it is hardcoded so we have to keep this in mind in order not to do some mistake in future

**Pull request merged**